File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/filtri/filtroFinalita2.ts
import { RegolamentoConcimazione } from "../metaschema/RegolamentoConcimazione";
import { GruppoFinalita } from "../metaschema/utilizzi/GruppoFinalita";
import { Specie } from "../metaschema/utilizzi/Specie";

export class FiltroFinalita2{
    specie: Specie;
    finalita: GruppoFinalita;
    regolamentoConcimazione: RegolamentoConcimazione;
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreDTOStd/InData/Metaschema/FiltroFinalita2.cs
using AgronicaCoreModelsSTD.metaschema;
using AgronicaCoreModelsSTD.metaschema.utilizzi;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreDTOStd.InData.Metaschema
{
    public class FiltroFinalita2
    {
        public Specie specie { get; set; }
        public GruppoFinalita finalita { get; set; }
        public RegolamentoConcimazione regolamentoConcimazione { get; set; }

    }
}
